src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
First of all, extract the common part `Colors.values()` to the variable.

Also, don't use enum `toString()`, instead use `name`. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution).

```suggestion
        Color[] colors = Color.values();
        int index = new Random().nextInt(colors.length);
        return colors[index].name();
```